jsonnet/thanos-mixin/alerts.yaml

```diff
@@ -60,6 +60,13 @@ groups:
       severity: warning
 - name: thanos-querier.rules
   rules:
+  - alert: ThanosQuerierIsNotRunning
+    annotations:
+      message: Thanos Querier is not running or just not scraped yet.
+    expr: up{job=~"thanos-querier.*"} == 0 or absent({job=~"thanos-querier.*"})
```

Comment:
This can even be `absent(up{job...})`

Reply:
Yes, we have had this helper for a while and it's served us well: https://github.com/kubernetes-monitoring/kubernetes-mixin/blob/master/alerts/absent_alerts.libsonnet